src/ansys/geometry/core/tools/pull_tools.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides tools for pulling geometry."""
+
+from typing import TYPE_CHECKING, List, Union
+
+from ansys.api.geometry.v0.commands_pb2 import ChamferRequest
+from ansys.api.geometry.v0.commands_pb2_grpc import CommandsStub
+from ansys.geometry.core.connection import GrpcClient
+from ansys.geometry.core.errors import protect_grpc
+from ansys.geometry.core.misc.checks import min_backend_version
+from ansys.geometry.core.typing import Real
+
+if TYPE_CHECKING:  # pragma: no cover
+    from ansys.geometry.core.designer.edge import Edge
+    from ansys.geometry.core.designer.face import Face
+
+
+class PullTools:
+    """Pull tools for PyAnsys Geometry.
```

Comment:
```suggestion
    """Provides pull tools for PyAnsys Geometry.
```
Class descriptions should begin with a verb ending in "s" or "es"